soccer/src/soccer/control/motion_control.cpp

```diff
@@ -74,6 +74,13 @@ MotionControl::MotionControl(int shell_id, rclcpp::Node* node)
         [this](PlayState::Msg::SharedPtr play_state_msg) {  // NOLINT
             play_state_ = rj_convert::convert_from_ros(*play_state_msg).state();
         });
+
+    error_x_pub_ =
+        node->create_publisher<std_msgs::msg::Float64>("motion_control/pose_error_x", 10);
```

Comment:
Let's prepend this name with debug (i.e. "debug/motion_control/pose_error_x"). Same for the below topics.